tools/yaml-templates/build-test-publish.yml

```diff
@@ -98,6 +98,14 @@ steps:
       targetFolder: '$(Build.ArtifactStagingDirectory)\CDNFeed\$(package.version)\js'
     displayName: 'Copy TeamsJS Content for CDN'
 
+  - task: CopyFiles@2
+    inputs:
+      Contents: |
+        packages\teams-js\utils\*.json
```

Comment:
>utils [](http://example.com/codeflow?start=26&length=5)

I'm a bit concerned about putting this json file into a "utils" directory, since it may take some people by surprise that files in that directory get published directly to a CDN (let alone all files with a .json extension). Perhaps consider a directory like `packages\teams-js\publishedToCdn`

Reply:
I like this idea, I will update this to reflect that